### PR TITLE
pass missing batchInterval if useBatchingInterface

### PR DESCRIPTION
**Pull request by comus**

### CODE REVIEW

**Comment by comus** [comment]
fix #78

another topic:

the atmospherejs published version of apollo 0.4.0 have some problems about graphiql.

someone described in #79

'/graphiql' hang on loading. just the ui keep loading. but git clone local copy is ok. i don't know why

**Comment by csillag** [comment]
> '/graphiql' hang on loading. just the ui keep loading.

To be more precise, the page lows, it's the queries that hang.

**Comment by xavcz** [comment]
Ok I'm having a look, thanks for reporting 🙆‍♂️

**Comment by xavcz** [comment]
Ok nailed it:

1. we import server-side `createMeteorNetworkInterface` & `meteorClientConfig`: https://github.com/apollographql/meteor-integration/blob/master/main-server.js#L14
2. `defaultClientConfig` has a `networkInterface` key with `createMeteorNetworkInterface()` value: this **executes** the function!! : https://github.com/apollographql/meteor-integration/blob/master/main-client.js#L101
3. `createMeteorNetworkInterface` fails because by default it's a batching interface, but at the moment it doesn't get the `batchingInterval` passed to the network interface options.

➡️ 0.4.0 fails silently
➡️ With @comus patch, it tells you that something is going weird: "howdy no batching interval set".

I'm on my way to push a patch to solve that, sorry for the inconvenience folks.

File: main-client.js [hunk 1]
```diff
@@ -44,6 +44,11 @@ export const createMeteorNetworkInterface = (customNetworkInterfaceConfig = {})
     uri: config.uri,
     opts: config.opts,
   }
+  
+  // http://dev.apollodata.com/core/network.html#BatchingExample
+  if (useBatchingInterface) {
+    interfaceArgument.batchInterval = config.batchInterval
```
Comment (xavcz) [inline main-client.js]: 🎉

**Pull request merged**